dotCMS/src/main/java/com/dotmarketing/util/RuntimeUtils.java

```diff
@@ -26,6 +27,24 @@ public class RuntimeUtils {
     public static final String ARM64_ARCH = "aarch64";
     public static final String X86_64_ARCH = "x86_64";
 
+
+
+    private static final Lazy<Boolean> ENABLE_LOGGING = Lazy.of(() -> Boolean.parseBoolean(System.getProperty("dotcms.runtimeutils.enablelogging", "false")));
```

Comment:
not sure if this is the desired thing, may be use Config?

Reply:
This will only read from the ENV configuration so it cannot be enabled from the back end.